README.md

```diff
@@ -1,7 +1,15 @@
 # StuDash
-Digital home learning environment. 
+</br>
+#StuDash
+"**Digital home learning environment**. 
+
 The environment is used for teaching, learning and assessment.
+
 The system makes it possible to illustrate, research and practice the material learned in a meaningful and in-depth way that is close to the students' language and increases curiosity and motivation for learning.
 StuDash includes many assessment assignments, study units, videos, simulations, presentations and more, all in a variety of subjects like: mathematics, science, Hebrew, English, history etc.
-The content is adapted to the curriculum of the Ministry of Education
-In addition, StuDash is an advanced and friendly learning environment that provides a complete and comprehensive systemic response also to the teachers.
+The content is adapted to the curriculum of the Ministry of Education!
+
+In addition, StuDash is an advanced and friendly learning environment that provides a complete and comprehensive systemic response also to the teachers."
+</br>
+## DEMO - Image
+[ofek App](https://www.easyform.co.il/SystemFiles/242/Ofek.jpg)
```

Comment:
Embedding images can be problematic legally. 
Please put a link to a web page on the site or remove it from here.

Reply:
I meant removing to removing the link to the png file, not removing the README